src/lib/common/sol-platform-impl-contiki.c

```diff
@@ -135,3 +135,31 @@ sol_platform_impl_umount(const char *mpoint, void (*cb)(void *data, const char *
     SOL_WRN("Not implemented");
     return -ENOTSUP;
 }
+
+int
+sol_plataform_unregister_hostname_monitor(void)
+{
+    SOL_WRN("Not implemented");
+    return -ENOTSUP;
```

Comment:
If it's unavailable, is it possible to generate one based on, say, the MAC address, or the machine ID, so it's always available, making it easier to obtain it (no error checking from the client-side, etc).